packages/frontend/src/pages/NewInstructLabSession.svelte

```diff
@@ -0,0 +1,260 @@
+                <Button
+                  title="Select knowledge file"
+                  type="link"
+                  disabled={trainingType !== 'knowledge'}
+                  class={trainingType !== 'knowledge'
+                    ? 'text-[var(--pd-input-field-disabled-text)] hover:bg-transparent '
+                    : ''}
+                  on:click={addKnowledge}
+                  icon={faPlusCircle}>Add knowledge to use</Button>
+                <span
+                  class:text-[var(--pd-input-field-disabled-text)]={trainingType !== 'knowledge'}
+                  class:text-[var(--pd-table-body-text)]={trainingType === 'knowledge'}>
+                  Add a YAML file downloaded from your Knowledge contribution on InstructLab. This will create a
+                  synthetic dataset based on the YAML file and retrain the model using this data.
+                  <Link on:click={openInstructLabDocumentation}>Learn more about knowledge contributions</Link>
+                </span>
+              </div>
+            </div>
+
+            <!-- skills -->
+            <div
+              class:border-2={true}
+              class="rounded-md p-5 cursor-pointer bg-[var(--pd-content-card-inset-bg)] flex flex-col gap-y-2"
+              aria-label="Skills"
+              aria-pressed={trainingType === 'skills' ? 'true' : 'false'}
+              class:border-[var(--pd-content-card-border-selected)]={trainingType === 'skills'}
+              class:border-[var(--pd-content-card-border)]={trainingType !== 'skills'}>
+              <button
+                title="Use Skills"
+                class="flex flex-row align-middle items-center"
+                onclick={setTrainingType.bind(undefined, 'skills')}>
+                <div
+                  class="text-2xl"
+                  class:text-[var(--pd-content-card-border-selected)]={trainingType === 'skills'}
+                  class:text-[var(--pd-content-card-border)]={trainingType !== 'skills'}>
+                  <Fa icon={faCircleCheck} />
+                </div>
+                <div
+                  class="pl-2"
+                  class:text-[var(--pd-content-card-text)]={trainingType === 'skills'}
+                  class:text-[var(--pd-input-field-disabled-text)]={trainingType !== 'skills'}>
+                  Add skills
+                </div>
+              </button>
+              <!-- files list -->
+              <div class="flex flex-col">
+                {#each skills_files as file (file)}
+                  <div
+                    class="bg-[var(--pd-label-bg)] text-[var(--pd-label-text)] max-w-full rounded-md px-2 py-1 mb-2 flex flex-row w-min h-min text-sm text-nowrap items-center">
+                    <Fa class="mr-2" icon={faFile} />
+                    <span class="overflow-x-hidden text-ellipsis max-w-full">
+                      {file}
+                    </span>
+                    <Button
+                      on:click={function (): void {
```

Comment:
wondering here why we need the function () and not give the removeSkills directly

Reply:
done, used binding